### PR TITLE
add contents for June 05 2023

### PR DESCRIPTION
**Pull request by axkirillov**

### CODE REVIEW

**Comment by kiran94** [comment]
Hey @phaazon, @axkirillov 

Could you advise what would be the best course of action if I also wanted to make a contribution for this round of news? :smile: 

Given that the current branch for both this repository and the fork would have restrictive permissions on who can push to them presumably

File: contents/2023/June/0-core-updates/0-HEADER.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# [Neovim Core updates](#core) {#core}
```
Comment (kiran94) [inline contents/2023/June/0-core-updates/0-HEADER.md:1]: I believe that the `day` also needs to part of the folder structure 

See May for an example https://github.com/phaazon/this-week-in-neovim-contents/tree/master/contents/2023/May

Comment (axkirillov) [reply contents/2023/June/0-core-updates/0-HEADER.md:1]: @kiran94 thanks, I will fix.

**Comment by axkirillov** [comment]
@kiran94 I am not sure. I guess you can fork my fork? I don't think the prs get adjusted across forks, so I have to target my own fork as well, when adding content. The other option is simply wait until this gets merged.

**Comment by malbertzard** [comment]
@kiran94 hey I have another pull request with a few plugins already added. (Including yours) Maybe checkout that PR and change the file if you want to. Or if i missed anything major.

**Pull request merged**